tools/inference_gradio.py

```diff
@@ -66,29 +72,44 @@ def get_args() -> Namespace:
 
     args = parser.parse_args()
 
+    # Model config is passed as the input argument else assume that model name is passed to model
+    if args.model.endswith(".yaml"):
+        args.model = Path(args.model)
+
     return args
 
 
-def get_inferencer(config_path: Path, weight_path: Path, meta_data_path: Path) -> Inferencer:
-    """Parse args and open inferencer."""
-    config = get_configurable_parameters(config_path)
+def get_inferencer(model: Union[Path, str], weight_path: Path, meta_data_path: Optional[Path] = None) -> Inferencer:
+    """Parse args and open inferencer.
+
+    Args:
+        config_path (Path): Path to model configuration file or the name of the model.
+        weight_path (Path): Path to model weights.
+        meta_data_path (Optional[Path], optional): Metadata is required for OpenVINO models. Defaults to None.
+
+    Raises:
+        ValueError: If unsupported model weight is passed.
+
+    Returns:
+        Inferencer: Torch or OpenVINO inferencer.
+    """
+    if isinstance(model, Path):
+        config = get_configurable_parameters(config_path=model)
+    else:
+        config = get_configurable_parameters(model_name=model)
     # Get the inferencer. We use .ckpt extension for Torch models and (onnx, bin)
     # for the openvino models.
     extension = weight_path.suffix
     inferencer: Inferencer
     if extension in (".ckpt"):
         module = import_module("anomalib.deploy.inferencers.torch")
         TorchInferencer = getattr(module, "TorchInferencer")  # pylint: disable=invalid-name
-        inferencer = TorchInferencer(
-            config=config, model_source=weight_path, meta_data_path=meta_data
-        )
```

Comment:
Is this needed to correct the parameters. If `TorchInferencer` and `OpenVINOInferencer` are explicitly imported, we wouldn't have to disable the pylint, no?